models/classes/LtiProvider/LtiProviderService.php

```diff
@@ -20,22 +20,27 @@
 
 namespace oat\taoLti\models\classes\LtiProvider;
 
+use oat\generis\model\OntologyAwareTrait;
 use oat\oatbox\service\ConfigurableService;
+use oat\taoDelivery\model\execution\DeliveryExecutionService;
```

Comment:
Please do not use `taoDelivery` and `taoDeliveryRdf` extensions, I think that can be bypassed by delegating responsibilities to `taoLtiConsumer`

Reply:
I cannot find any other service that suits my purpose. I see `taoDelivery` as required dependency. I don't how I can bypass it by anything else.

Reply:
@bartlomiejmarszal you can remove the method `searchByDeliveryExecutionId` which is not necessary here, but only in the taoConsumer extension, so you remove those dependencies :)